Firestore/Source/Local/FSTLevelDB.mm

```diff
@@ -229,9 +225,9 @@ - (LevelDbTransaction *)currentTransaction {
   return [[FSTLevelDBRemoteDocumentCache alloc] initWithDB:self serializer:self.serializer];
 }
 
-- (void)startTransaction {
+- (void)startTransaction:(const std::string &)label {
```

Comment:
The type here should be `absl::string_view` to avoid copying all the arguments to `startTransaction` into a temporary string only to copy again for the constructor to `LevelDbTransaction`.

I wish there were a way to restrict the argument to a compile-time constant string literal (then we could avoid copies altogether) but I can't see any clean way to do it.

Reply:
Done.